2-copy-of-code/lesson-13/scripts/amazon.js

```diff
@@ -72,6 +72,7 @@ document.querySelectorAll('.js-add-to-cart')
       const quantitySelector = document.querySelector(
         `.js-quantity-selector-${productId}`
```

Comment:
Instead of 'productId'
Use the 'button.dataset.productId'

Reply:
[connect me with instagram to solve your issue](https://www.instagram.com/banda.shri/)